next.clayui.com/content/docs/css/components/forms/date-and-time-pickers.md

```diff
@@ -0,0 +1,641 @@
+					<button aria-label="2019 02 10" class="date-picker-date date-picker-calendar-item" type="button">10</button>
+					<button aria-label="2019 02 11" class="date-picker-date date-picker-calendar-item" type="button">11</button>
+					<button aria-label="2019 02 12" class="date-picker-date date-picker-calendar-item" type="button">12</button>
+					<button aria-label="2019 02 13" class="date-picker-date date-picker-calendar-item active" tabindex="-1" type="button">13</button>
+					<button aria-label="2019 02 14" class="date-picker-date date-picker-calendar-item" type="button">14</button>
+					<button aria-label="2019 02 15" class="date-picker-date date-picker-calendar-item" type="button">15</button>
+					<button aria-label="2019 02 16" class="date-picker-date date-picker-calendar-item" type="button">16</button>
+				</div>
+				<div class="date-picker-date-row date-picker-row">
+					<button aria-label="2019 02 17" class="date-picker-date date-picker-calendar-item" type="button">17</button>
+					<button aria-label="2019 02 18" class="date-picker-date date-picker-calendar-item" type="button">18</button>
+					<button aria-label="2019 02 19" class="date-picker-date date-picker-calendar-item" type="button">19</button>
+					<button aria-label="2019 02 20" class="date-picker-date date-picker-calendar-item" type="button">20</button>
+					<button aria-label="2019 02 21" class="date-picker-date date-picker-calendar-item" type="button">21</button>
+					<button aria-label="2019 02 22" class="date-picker-date date-picker-calendar-item" type="button">22</button>
+					<button aria-label="2019 02 23" class="date-picker-date date-picker-calendar-item" type="button">23</button>
+				</div>
+				<div class="date-picker-date-row date-picker-row">
+					<button aria-label="2019 02 24" class="date-picker-date date-picker-calendar-item" type="button">24</button>
+					<button aria-label="2019 02 25" class="date-picker-date date-picker-calendar-item" type="button">25</button>
+					<button aria-label="2019 02 26" class="date-picker-date date-picker-calendar-item" type="button">26</button>
+					<button aria-label="2019 02 27" class="date-picker-date date-picker-calendar-item" type="button">27</button>
+					<button aria-label="2019 02 28" class="date-picker-date date-picker-calendar-item" type="button">28</button>
+					<button aria-label="2019 03 01" class="active date-picker-date date-picker-calendar-item next-month-date" type="button">1</button>
+					<button aria-label="2019 03 02" class="date-picker-date date-picker-calendar-item next-month-date" type="button">2</button>
+				</div>
+			</div>
+		</div>
+	</div>
+</div>
+```
+
+## Time Picker
+
+<div class="sheet-example">
+	<div class="col-md-5">
+		<div class="input-group">
+			<div class="input-group-item">
+				<input class="form-control" name="timer" type="time" value="00:00">
+			</div>
+			<div class="input-group-item input-group-item-shrink">
+				<span class="input-group-text">(GMT+01:00)</span>
```

Comment:
hey @pat270 have any class we can change to not add a gray background here? when the Time Picker is outside the Date Picker.